File: AllSamples/Samples/Samples_Source/indigo/remoteobjects/remoteobjects.hellonewremoteobject_vb/server/host.vb
Imports System
Imports System.MessageBus
Imports System.MessageBus.Remoting
Namespace Microsoft.Samples.MessageBus.Quickstarts.RemoteObjects
    Public Class Host
        Public Shared Sub Main(ByVal args() As String)
            ' The remoting configuration starts with the 
            ' System.MessageBus.ServiceEnvironment class.
            Dim se As ServiceEnvironment = ServiceEnvironment.Load()
            ' Start the service environment.
            se.Open()

            Console.WriteLine("Listening for requests. Press Enter to exit...")
            Console.ReadLine()

            ' Close the environment.
            se.Close()
        End Sub
    End Class
End Namespace

File: AllSamples/Samples/Samples_Source/indigo/remoteobjects/remoteobjects.hellonewremoteobject_cs/server/host.cs
using System; 
using System.MessageBus;
using System.MessageBus.Remoting;

namespace Microsoft.Samples.MessageBus.Quickstarts.RemoteObjects
{
    public class Host
    {
        public static void Main()
        {
            // The remoting configuration starts with the 
            // System.MessageBus.ServiceEnvironment class.
            ServiceEnvironment se = ServiceEnvironment.Load();

            // Start the ServiceEnvironment. 
            se.Open();

            Console.WriteLine("Listening for requests. Press Enter to exit...");
            Console.ReadLine();

            // Close the environment.
            se.Close();
        }
    }
}